### DISCUSSION TITLE
Use @vitejs/plugin-react-swc?

### DISCUSSION THREAD

**Discussion by ianschmitz** [storybookjs/builder-vite#529] [Ideas]
The Vite team released a new React plugin that replaces babel with SWC during development.

See: 

https://vitejs.dev/blog/announcing-vite4.html#new-react-plugin-using-swc-during-development
https://github.com/vitejs/vite-plugin-react-swc

Would you consider moving this package to that plugin? This avoids consumers having to install both `@vitejs/plugin-react` (currently a dependency of this package) and `@vitejs/plugin-react-swc` in their project if they choose to use the SWC variant in Vite.

**Comment by IanVS** [comment]
Storybook 7.0 uses your vite.config.js automatically. Would you be interested in trying it out with the SWC plugin and seeing how it works?  You can run `npx sb@next upgrade --prerelease` to upgrade.
